### ISSUE TITLE
Introduce a new boost thread library version

### ISSUE THREAD

**Issue by redlicha** [openvstorage/volumedriver-buildtools#17]
https://github.com/openvstorage/volumedriver/issues/268 requires fixes in boost::thread (futures) -
 https://svn.boost.org/trac10/ticket/11231 . Upgrading boost wholesale to 1.62 breaks backward compatibility as boost::serialization bumped archive versions.

**Comment by wimpers** [comment]
@redlicha how urgent is this? Did you pick this up when working on the pipelining of the DTL. Please advice.

**Comment by redlicha** [comment]
@wimpers, it's indeed a prerequisite of the DTL pipelining. It's currently implemented on the feature/async-dtl-writes branch and is to be merged into the buildtools' dev branch once the corresponding volumedriver feature branch is merged - https://github.com/openvstorage/volumedriver/pull/334

**Comment by wimpers** [comment]
@redlicha what is the state of this one? It is merged in 7.x . can this one be closed?

**Comment by redlicha** [comment]
Closing this one (implemented on ee)
